### PR TITLE
Fwdport/7.4 beta master

### PR DESCRIPTION
**Pull request by alexanderchan-scality**
Integration: https://github.com/scality/Integration/pull/656
Federation: https://github.com/scality/Federation/pull/1419
Utapi: https://github.com/scality/utapi/pull/170

### CODE REVIEW

**Comment by alexanderchan-scality** [comment]
@ironman-machine 
DEFAULT_BRANCH: master
SCALITY_FEDERATION_BRANCH: fwdport/7.4-beta-master
SCALITY_INTEGRATION_BRANCH: fwdport/7.4-beta-master
try

**Comment by alexanderchan-scality** [comment]
@ironman-machine 
DEFAULT_BRANCH: master

**Comment by alexanderchan-scality** [comment]
@ironman-machine clean_env

**Comment by alexanderchan-scality** [comment]
@ironman-machine 
SCALITY_FEDERATION_BRANCH=fwdport/7.4-beta-master

**Comment by alexanderchan-scality** [comment]
@ironman-machine 
SCALITY_INTEGRATION_BRANCH=fwdport/7.4-beta-master
DEFAULT_BRANCH=master

**Comment by alexanderchan-scality** [comment]
@ironman-machine try

File: package.json [hunk 1]
```diff
@@ -20,16 +20,17 @@
   "homepage": "https://github.com/scality/S3#readme",
   "dependencies": {
     "aws-sdk": "2.28.0",
-    "arsenal": "scality/Arsenal",
+    "arsenal": "scality/Arsenal#fwd/7.4-beta-master",
```
Comment (DavidPineauScality) [inline package.json]: Branch spec needs to be removed for master

File: package.json [hunk 2]
```diff
@@ -20,16 +20,17 @@
...
     "azure-storage": "^2.1.0",
+    "backo": "^1.1.0",
     "bucketclient": "scality/bucketclient",
     "commander": "^2.9.0",
+    "google-auto-auth": "^0.9.1",
     "node-uuid": "^1.4.3",
     "npm-run-all": "~4.0.2",
     "sproxydclient": "scality/sproxydclient",
-    "utapi": "scality/utapi",
+    "utapi": "scality/utapi#fwdport/7.4-beta-master",
```
Comment (DavidPineauScality) [inline package.json]: Branch spec needs to be removed for master

**Comment by alexanderchan-scality** [comment]
@ironman-machine clean_env

**Comment by alexanderchan-scality** [comment]
@ironman-machine 
SCALITY_METADATA_BRANCH=fwdport/7.4-beta-master
SCALITY_S3_BRANCH=fwdport/7.4-beta-master
SCALITY_FEDERATION_BRANCH=fwdport/7.4-beta-master
SCALITY_INTEGRATION_BRANCH=fwdport/7.4-beta-master
SCALITY_IDENTISEE_BRANCH=fwdport/7.4-beta-master
SCALITY_UTAPI_BRANCH=fwdport/7.4-beta-master
SCALITY_BACKBEAT_BRANCH=fwdport/7.4-beta-master
SCALITY_S3BROWSER_BRANCH=fwdport/7.4-beta-master

**Comment by alexanderchan-scality** [comment]
@ironman-machine try

**Comment by alexanderchan-scality** [comment]
passing integration: http://ci.ironmann.io/gh/scality/Integration/20583

**Comment by alexanderchan-scality** [comment]
@ironman-machine clean_env

**Comment by alexanderchan-scality** [comment]
@ironman-machine
SCALITY_METADATA_BRANCH=fwdport/7.4-beta-master
SCALITY_S3_BRANCH=fwdport/7.4-beta-master
SCALITY_FEDERATION_BRANCH=fwdport/7.4-beta-master
SCALITY_INTEGRATION_BRANCH=fwdport/7.4-beta-master
SCALITY_IDENTISEE_BRANCH=fwdport/7.4-beta-master
SCALITY_UTAPI_BRANCH=fwdport/7.4-beta-master
SCALITY_BACKBEAT_BRANCH=fwdport/7.4-beta-master
SCALITY_S3BROWSER_BRANCH=fwdport/7.4-beta-master
try

**Comment by alexanderchan-scality** [comment]
@ironman-machine clean_env

**Comment by alexanderchan-scality** [comment]
@ironman-machine try

**Comment by alexanderchan-scality** [comment]
@ironman-machine clean_env

**Comment by alexanderchan-scality** [comment]
@ironman-machine
SCALITY_S3_BRANCH=fwdport/7.4-beta-master
SCALITY_FEDERATION_BRANCH=fwdport/7.4-beta-master
SCALITY_INTEGRATION_BRANCH=fwdport/7.4-beta-master
SCALITY_BACKBEAT_BRANCH=fwdport/7.4-beta-master
try

**Comment by alexanderchan-scality** [comment]
@ironman-machine try

**Comment by alexanderchan-scality** [comment]
E2E passing: http://ci.ironmann.io/gh/scality/Integration/20746

**Comment by alexanderchan-scality** [comment]
@ironman-machine clean_env

**Comment by alexanderchan-scality** [comment]
@ironman-machine
SCALITY_S3_BRANCH=fwdport/7.4-beta-master
SCALITY_FEDERATION_BRANCH=fwdport/7.4-beta-master
SCALITY_INTEGRATION_BRANCH=fwdport/7.4-beta-master
SCALITY_BACKBEAT_BRANCH=fwdport/7.4-beta-master
CI_AGGRESSOR_FORCE_BUILD=true
RUN_BACKBEAT_CRR_TESTS=true
try

**Comment by rahulreddy** [comment]
Integration passed here http://ci.ironmann.io/gh/scality/Integration/20791

**Pull request merged**